tools/gitserver/Dockerfile

```diff
@@ -1,7 +1,6 @@
 FROM alpine:3.12
 
-RUN apk update &&\
```

Comment:
is there a reason to not update the package list?

Reply:
we are updating the packages with `--no-cache` flag, just not storing it.